src/query.rs

```diff
@@ -754,6 +778,68 @@ mod tests {
         assert!(sat.CPUScore >= 0.0);
     }
 
+    #[test]
+    fn it_can_query_quotes_and_slashes() {
+        let wmi_con = wmi_con();
+
+        #[derive(Deserialize, Debug)]
+        struct Win32_Service {
+            Name: String,
+
+            /// Name="LSM" (among other services?) lacks a PathName
+            /// PathName is a great source of quote marks and back slashes surounding paths into "Program Files" to test escaping against
+            PathName: Option<String>,
+        }
+
+        let services = wmi_con.query::<Win32_Service>().unwrap();
+        assert!(services.len() >= 1);
+
+        // Checking every service with `\"` and `\\` in their PathName s would be too slow (60+ seconds!), so just check one
+        let service = services.iter().find(|service| service.PathName.as_ref().map_or(false, |path_name| path_name.contains("\"") && path_name.contains("\\"))).unwrap();
+
+        let mut filters = HashMap::new();
+        filters.insert(String::from("Name"), FilterValue::String(service.Name.clone()));
+        filters.insert(String::from("PathName"), FilterValue::String(service.PathName.as_ref().unwrap().clone()));
+        let services_filtered = wmi_con.filtered_query::<Win32_Service>(&filters).unwrap();
+
+        assert!(services_filtered.len() >= 1, "Unable to find service {:?} via filters", service.Name);
+    }
+
+    #[test]
+    fn it_can_query_unicode() {
```

Comment:
IMO we don't need to test this, because we only verify that WMI/COM support utf16.
Also because we have no way to guarantee that a suitable `Win32_Product` exists, this test is probably an overkill :)